README.md

```diff
@@ -1,20 +1,65 @@
 
-Python package integrating Jupyter notebooks with various graph-stores including
-[Apache TinkerPop](https://tinkerpop.apache.org/) and [RDF SPARQL](https://www.w3.org/TR/rdf-sparql-query/).
+The graph notebook provides an easy way to interact with graph databases using Jupyter notebooks. Using this open-source Python package, you can connect to any graph database that supports the [Apache TinkerPop](https://tinkerpop.apache.org/) or the [RDF SPARQL](https://www.w3.org/TR/rdf-sparql-query/) graph model. These databases could be running locally on your desktop or in the cloud. Graph databases can be used to explore a variety of use cases including [knowledge graphs](https://aws.amazon.com/neptune/knowledge-graphs-on-aws/) and [identity graphs](https://aws.amazon.com/neptune/identity-graphs-on-aws/).
 
-## Requirements
+### Visualizing Gremlin queries:
 
-- Python 3.6.1 or higher, Python 3.7
-- Jupyter Notebook
+![Gremlin query and graph](./images/GremlinQueryGraph.png)
 
-## Introduction
-The graph-notebook provides a way to interact using a Jupyter notebook with any graph database that follows the Gremlin Server or RDF HTTP protocols. These databases could be running locally on your laptop, in a private data center or in the cloud. This project was initially created as a way to work with Amazon Neptune but is not limited to that database engine. For example you can connect to a Gremlin Server running on your laptop using this solution. The instructions below describe the process for connecting to Amazon Neptune. We encourage others to contribute configurations they find useful. There is an [`additional-databases`](additional-databases) folder where such information can be found. 
+### Visualizing SPARQL queries:
+
+![SPARL query and graph](./images/SPARQLQueryGraph.png)
+
+Instructions for connecting to the following graph databases:
+
+|             Endpoint            |       Graph model       |   Query language    |
+| :-----------------------------: | :---------------------: | :-----------------: | 
+|[Gremlin Server](#gremlin-server)|     property graph      |       Gremlin       |
+|    [Blazegraph](#blazegraph)    |            RDF          |       SPARQL        |
+|[Amazon Neptune](#amazon-neptune)|  property graph or RDF  |  Gremlin or SPARQL  |
+
+We encourage others to contribute configurations they find useful. There is an [`additional-databases`](https://github.com/aws/graph-notebook/blob/main/additional-databases) folder where more information can be found.
+
+## Features
+
+#### Notebook cell 'magic' extensions in the IPython 3 kernel
+`%%sparql` - Executes a SPARQL query against your configured database endpoint.
+
+`%%gremlin` - Executes a Gremlin query against your database using web sockets. The results are similar to what the Gremlin console would return.
+
+**TIP** :point_right:  There is syntax highlighting for both `%%sparql` and `%%gremlin` queries to help you write good code.
```

Comment:
Maybe something like 

**TIP** :point_right:  There is syntax highlighting for both `%%sparql` and `%%gremlin` queries to help you structure your queries more easily.

Reply:
Good point! Will change.